radio/lib/single-selection-controller.ts

```diff
@@ -219,6 +219,10 @@ export class SingleSelectionController implements ReactiveController {
       nextSibling.checked = true;
```

Comment:
Does calling `nextSibling.click()` work instead of dispatching an event? 🤔

```ts
nextSibling.removeAttribute('tabindex');
nextSibling.click();
break;
```

If so that could simplify this a bit!

Reply:
Yes, that makes sense